### ISSUE TITLE
Broadcast failed on notarize when multiple tests

### ISSUE THREAD

**Issue by jpbeaudet** [shayanb/keystamp-crypto#2]

**Comment by shayanb** [comment]
Yeah this issue is because of the bitcoin input management, for now notarize has 2 inputs to use and have to wait for them to confirm to be able to do more notarization. for production level, I should make it more efficient to have more inputs ready to go, or use unconfirmed inputs. but the latter might have some bad implications that I don't want to go down that road.
